brink/the-bitcoin-development-podcast/code-review-and-bip324.md

```diff
@@ -203,47 +208,48 @@ If that option is turned on, they're just really created on the go, whenever a n
 And another thing which uses that is the ScanBlocks RPC by James O'Byrne.
 I'm always not sure how to pronounce his last name.
 I think that one came in one release earlier, if I'm not mistaken, 24.
-And that one also, that is an RPC call though, and it also, you can pass a set of output scripts that you're interested in and it would just return you all the blocks that are matching.
+That is an RPC call though, and also, you can pass a set of output scripts that you're interested in and it would just return you all the blocks that are matching.
 So it's kind of nice that we can use that to make the the life of users easier, hopefully.
 
 ## The scarcity and importance of code review
 
 Mike Schmidt: 00:17:05
 
-One thing that I got feedback on about your work is from a prominent Bitcoin Core dev who said, quote, the stacks review matters.
+One thing that I got feedback on about your work is from a prominent Bitcoin Core dev who said, quote, "the stacks review matters."
 The stacks review is valuable, something along those lines.
-And so I think you've done quite a bit of review and maybe You can comment on that in the context of How do you think about how much time you spend authoring new code and PRs versus review?
+And so I think you've done quite a bit of review and maybe you can comment on that in the context of how do you think about how much time you spend authoring new code and PRs versus review?
 And why do you think that this person said that your review counts?
 
 Sebastian Falbesoner: 00:17:47
 
 To your first question, I think the review time I spend is more than authoring time.
-I would even love to spend more time reviewing, because I think it's the most, it's the resource that is most missing in Bitcoin Core actually.
+I would even love to spend more time reviewing, because I think it's the resource that is most missing in Bitcoin Core actually.
 Like it would be great if we have more review power.
-And yeah, according to what person's reviews counts, I think it's also kind of a proof of work system in Bitcoin Core, Not specifically regarding me, but in general, if you over the time give review comments that are helpful or are considered helpful by others, then of course you're getting taken more seriously.
-Seriously and what I, I think my review style has been also inspired by other people from the top of my head.
-I could name Ross Janowski who give.
-Very great detailed review comments and that's inspired someone, right?
+And yeah, according to what person's reviews counts, I think it's also kind of a proof of work system in Bitcoin Core.
+Not specifically regarding me, but in general, if you over the time give review comments that are helpful or are considered helpful by others, then of course you're getting taken more seriously.
+I think my review style has been also inspired by other people from the top of my head.
+I could name Russ yanofsky who give very great detailed review comments and that's inspired someone, right?
```

Comment:
Potential places that you can look at when trying to identify and fix a name:
- https://btctranscripts.com/speakers
- search for the name on the repo of the project, e.g https://github.com/bitcoin/bitcoin